modules/admin_manual/pages/configuration/server/background_jobs_configuration.adoc

```diff
@@ -118,13 +118,19 @@ If used, these should be scheduled to run on a daily basis.
 
 While not exhaustive, these include:
 
-==== CleanupChunks
+* xref:cleanupchunks[CleanupChunks]
```

Comment:
These, technically speaking, shouldn't likely be in this commit. However, the changed seemed worthwhile as a small content cleanup ([the Boy Scout Rule, as Uncle Bob says](https://medium.com/@biratkirat/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385)).